Some fixes to the OBI keycloak theme

danifr opened the pull request:
- OBI logo is now a link to: https://www.openbraininstitute.org
- Fixed size of OBI logo
- Moved `<style>` block in template to .css file

The change is now deployed in **staging**
cc: @ayimaok

danifr committed:
Add missing position absolute

danifr committed:
Move styles to css file

danifr committed:
Add link to OBI logo in keycloak theme

pgetta commented on login/login.ftl:
```diff
@@ -8,18 +8,12 @@
   <div class="page-container">
     <div class="login-header">
-      <img class="icon" src="${url.resourcesPath}/img/obi_logo.svg" alt="OBI logo">
+      <a href="https://www.openbraininstitute.org/"><img class="icon" src="${url.resourcesPath}/img/obi_logo.svg" alt="OBI logo"></a>
```
Later on probably we can parameterise the link, so that in in staging it will point back to staging, but it's good for now.

danifr replied:
I agree. I am not sure how to make the theme aware in what environment is being deployed on though. Any ideas?
Is there a way in simple HTML I can point to the rootURL?

DriesVerachtert replied:
It could simply be a link to "/", which is the root of whatever domain name is used at that point.

In staging, when using staging.openbraininstitute.org/auth, that would point to staging.openbraininstitute.org
In prod, when using www.openbraininstitute.org/auth, that would point to www.openbraininstitute.org

danifr replied:
thanks I think that would work. I will test it in staging 1st

danifr committed:
Replace svg code by the same one core-web-app uses

This one https://github.com/openbraininstitute/core-web-app/blob/2025.03.28.1/src/components/logo/as-svg.tsx

danifr committed:
Fix OBI log top and left CSS values
